doc/pilight-flash.1

```diff
@@ -0,0 +1,66 @@
+.TH pilight-flash 1 "10 July 2017" "7.0-dev" "pilight 7.0-dev"
+.SH NAME
+pilight-flash \- Flash hardware with pilight firmware
+.SH SYNOPSIS
+.B pilight-flash \fI--file FIRMWARE_FILE\fR
+.br
+.B pilight-control [\fIOPTION\fR]... \fI--file FIRMWARE_FILE\fR
+.SH DESCRIPTION
+.B pilight-control
+is a tool to flash pilight firmware to hardware such as ATTiny or ATMega 
```

Comment:
ATTiny is a subtype created by the brand ATMel. So this should say.
`such as ATMel chips (such as the ATTiny and ATMega)`